marge/bot.py

```diff
@@ -153,15 +153,21 @@ def _process_merge_requests(self, repo_manager, project, merge_requests):
                 log.exception('BatchMergeJob failed: %s', err)
         log.info('Attempting to merge the oldest MR...')
         merge_request = merge_requests[0]
-        merge_job = single_merge_job.SingleMergeJob(
+        merge_job = self._get_job(
+            project=project, merge_request=merge_request, repo=repo,
+            options=self._config.merge_opts,
+        )
+        merge_job.execute()
+
+    def _get_job(self, project, merge_request, repo, options):
+        return single_merge_job.SingleMergeJob(
             api=self._api,
             user=self.user,
             project=project,
             merge_request=merge_request,
             repo=repo,
             options=self._config.merge_opts,
```

Comment:
This should probably be `options` given you're passing it to this method; otherwise there's no point passing it down if you don't need it to be different.

Reply:
You are totally right : options=options